src/Pages/SignUp.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import styled from "styled-components";
+import Register from "../components/auth/Register";
+
+const Styles = styled.div``;
+
+const SignUp = () => {
+    return (
+        <Styles>
+            <Register />
+        </Styles>
+    );
+};
+
+export default SignUp;
```

Comment:
This file's indentation is 4 spaces. The rest of the project is 2 spaces. Idc which standard we go with but we should be consistent across all the files.